clang/lib/StaticAnalyzer/Checkers/MallocChecker.cpp

```diff
@@ -3867,8 +3892,8 @@ namespace clang {
 namespace ento {
 namespace allocation_state {
 
-ProgramStateRef
-markReleased(ProgramStateRef State, SymbolRef Sym, const Expr *Origin) {
+ProgramStateRef markReleased(ProgramStateRef State, SymbolRef Sym,
+                             const Expr *Origin) {
```

Comment:
This and similar hunks appear to be clang-format changes. We usually only format the lines we touch.
Could you please revert the non relevant hunks from the diff?